drivers/sensor/bosch/bmp388/bmp388.c

```diff
@@ -257,12 +262,8 @@ static int bmp388_sample_fetch(const struct device *dev,
 	}
 
 	/* convert samples to 32bit values */
-	bmp388->sample.press = (uint32_t)raw[0] |
-			       ((uint32_t)raw[1] << 8) |
-			       ((uint32_t)raw[2] << 16);
-	bmp388->sample.raw_temp = (uint32_t)raw[3] |
-				  ((uint32_t)raw[4] << 8) |
-				  ((uint32_t)raw[5] << 16);
+	bmp388->sample.press = sys_get_le24(&raw[0]);
```

Comment:
I'd change the var everywhere to bmp3xx.

Reply:
yes, can do this.